src/main/kotlin/ro/hibyte/betting/exceptions/types/EventTemplateNotFoundException.kt

```diff
@@ -0,0 +1,3 @@
+package ro.hibyte.betting.exceptions.types
+
+class EventTemplateNotFoundException(id: Long): RuntimeException("Event template with id=${id} was not found")
```

Comment:
I would get rid of `EventTemplateNotFoundException` and `BetTemplateNotFoundException` (and other not found exceptions) and create a new exception class that can be used for any entity type: `EntityNotFoundException(type: String, id: Long)`, where type would be something like *"bet type"* or *"bet template"*.

Reply:
Done.